src/frontend/app/shared/components/list/list-types/services-wall/service-instance-card/service-instance-card.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
+import { IServiceInstance } from '../../../../../../core/cf-api-svc.types';
+import { ServicesWallService } from '../../../../../../features/services/services/services-wall.service';
+import { DeleteServiceBinding, DeleteServiceInstance } from '../../../../../../store/actions/service-instances.actions';
+import { AppState } from '../../../../../../store/app-state';
+import { APIResource } from '../../../../../../store/types/api.types';
+import { AppChip } from '../../../../chips/chips.component';
+import { MetaCardMenuItem } from '../../../list-cards/meta-card/meta-card-base/meta-card.component';
+import { CardCell } from '../../../list.types';
+
+@Component({
+  selector: 'app-service-instance-card',
+  templateUrl: './service-instance-card.component.html',
+  styleUrls: ['./service-instance-card.component.scss'],
+  providers: [
+    ServicesWallService
+  ]
+})
+export class ServiceInstanceCardComponent extends CardCell<APIResource> implements OnInit {
+  cfGuid: string;
+  cardMenu: MetaCardMenuItem[];
+
+  @Input('row') row: APIResource<IServiceInstance>;
+
+  serviceInstanceTags: AppChip[];
+  hasMultipleBindings = new BehaviorSubject(true);
+
+
+
+  constructor(private store: Store<AppState>, private servicesWallService: ServicesWallService) {
+    super();
+
+    this.cardMenu = [
+      {
+        label: 'Detach',
+        action: this.detach,
+        disabled: this.hasMultipleBindings
+      },
+      {
+        label: 'Delete',
+        action: this.delete
+      }
+    ];
+
+  }
+
+  ngOnInit() {
+
+    this.serviceInstanceTags = this.row.entity.tags.map(t => ({
+      value: t
+    }));
+
+    this.cfGuid = this.row.entity.cfGuid;
+    this.hasMultipleBindings.next(!(this.row.entity.service_bindings.length > 0));
+
+  }
+
+
+  detach = () => {
+    // Only deletes the first binding, if more than one exist this should take user to the appropriate view
+    if (this.row.entity.service_bindings.length === 1) {
+      this.store.dispatch(new DeleteServiceBinding(
```

Comment:
On detatch (card + table view), the applications attached value/cell fails to update. This could be the same issue we've seen in other places where we need to manually set the row with a custom set.  There's a few places that use 
```
  private _row: T;
  @Input('row')
  get row() { return this._row; }
  set row(row: T) {
    this._row = row;
    if (row) {
      this.setValue(row);
    }
  }
```
or some just watch the entity itself

```
const fetchData$ = Observable.combineLatest(
      userRole$,
      this.cfEndpointService.getAppsInOrg(this.row)
    ).pipe(
      tap(([role, apps]) => {
        this.setValues(role, apps);
      })
      );
```

Reply:
Think I need more clarity on the proposed solutions. Do I need to refetch all the data manually after an update?